src/lib/core/managers/download.svelte.ts

```
@@ -0,0 +1,249 @@
+          this.afterInstallCleanup(item.id);
+        }
+      });
+    } catch (error) {
+      throw new LegendaryError('TODO');
+    }
+  }
+
+  async installOrUpdate(appId: string, callbacks: DownloadCallbacks = {}) {
+    const settings = await DataStorage.getDownloaderFile();
+    const streamId = `install_${appId}_${Date.now()}`;
+    const args = ['install', appId, '-y', '--skip-sdl', '--skip-dlcs', '--base-path', settings.downloadPath!];
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)